### DISCUSSION TITLE
Adding Interactivity to GrapeJS components

### DISCUSSION THREAD

**Discussion by 99bits** [GrapesJS/grapesjs#5456] [Q&A]
Hi friends,

GrapeJS is an awesome project and we would like to know if it's possible to add interactivity, for example, a sliding animation from left to right.  We tried AlpineJS - https://alpinejs.dev/ and HTMX- https://htmx.org/ but they don't seem to play well with GrapeJs.  Perhaps we are missing something.  Has anyone been able to add  interactivity to GrapeJS?  

Cheers.

**Comment by artf** [comment]
Can you share a reproducible demo of what you have been trying to do with alpinejs/htmx?

**Comment by andrescevp** [comment]
Check here: https://github.com/GrapesJS/grapesjs/discussions/5445
